plugins/generator-1.16.4/forge-1.16.4/triggers/command_executed.java.ftl

```diff
@@ -0,0 +1,18 @@
+@SubscribeEvent public void onCommand(CommandEvent event) {
+	Entity entity = event.getParseResults().getContext().getSource().getEntity();
+	if (entity != null) {
+		double i = entity.getPosition().getX();

```

Comment:
Use getPos(X/Y/Z) instead getPosition().get# of as it uses double precision, currently, you use int and loose location precision.

Reply:
done!